frontend/src/features/admin-form/preview/PreviewFormProvider.tsx

```diff
@@ -209,6 +212,9 @@ export const PreviewFormProvider = ({
         isAuthRequired,
         expiryInMs,
         isLoading,
+        isMobileSectionSidebarOpen,
+        handleMobileSectionSidebarOpen,
+        handleMobileSectionSidebarClose,
```

Comment:
Is this really needed? As in, wouldn't `...commonFormValues` supply the new args transparently, and so stating the new fields in this file (both line 60-62 and here) is unnecessary?

Reply:
you're right, not needed!